### PR TITLE
Support skipping resources diff

### LINKED ISSUE #257: Add support for hiding resource in diff
We have some resources that are big and always change. And we want to hide them in diff.

I guess the easiest way will be introducing some label or annotation and skipping entire resource if this annotation exists

### PR DESCRIPTION
**Pull request by r3nic1e**
Closes #257

### CODE REVIEW

**Comment by derbeneviv** [comment]
@r3nic1e maybe consider also ignorinb helm v2 annotations?  
https://helm.sh/docs/topics/chart_tests/
```
Note that until Helm v3, the job definition needed to contain one of these helm test hook annotations: helm.sh/hook: test-success or helm.sh/hook: test-failure. helm.sh/hook: test-success is still accepted as a backwards-compatible alternative to helm.sh/hook: test.
```

**Comment by r3nic1e** [comment]
@derbeneviv I don't think so - we have helm v3 embedded. So the second version of helm chart schema is implied

**Comment by derbeneviv** [comment]
@r3nic1e as i see in docs - `HookEvent.Hooktest` matches only `test` string

**Comment by derbeneviv** [comment]
I can confirm that all tests are now hidden for both helm3 and helm2 variants

**Review by zhilyaev — APPROVED** [review]

File: pkg/plan/diff.go [hunk 1]
```diff
@@ -11,11 +11,20 @@ import (
...
-// ErrPlansAreTheSame is returned when trying to compare plan with itself.
-var ErrPlansAreTheSame = errors.New("plan1 and plan2 are the same")
+var (
+	// ErrPlansAreTheSame is returned when trying to compare plan with itself.
+	ErrPlansAreTheSame = errors.New("plan1 and plan2 are the same")
+
+	// SkippedAnnotations is a map with all annotations to be skipped by differ.
+	SkippedAnnotations = map[string]string{
+		live.HookAnnotation:      string(live.HookTest),
+		"helmwave.dev/skip-diff": "true",
```
[the remark this answers is not in the document]
Comment (r3nic1e) [reply pkg/plan/diff.go]: fixed

**Review by zhilyaev — APPROVED** [review]

**Pull request merged**